public/en/index.html

```diff
@@ -66,7 +66,7 @@
                     <li class="gender-chart-label" data-gender="f">
                         <span class="fa-li"><i class="fas fa-lg fa-venus"></i></span>
                         Female:
-                        <span class="gender-chart-count"></span>
+                        <span class="gender-chart-count"></span></span>
```

Comment:
There is a `</span>` too much !
```suggestion
                        <span class="gender-chart-count"></span>
```